rpcs3/rpcs3qt/rpcn_settings_dialog.cpp

```diff
@@ -945,6 +966,20 @@ rpcn_friends_dialog::rpcn_friends_dialog(QWidget* parent)
 		add_update_list(m_lst_blocks, QString::fromStdString(blck), m_red_icon, QVariant(false));
 	}
 
+	auto history = np::load_players_history();
+	std::map<u64, std::string, std::greater<u64>> sorted_history;
+
+	for (const auto& [username, user_info] : history)
+	{
+		if (!data.friends.contains(username) && !data.requests_sent.contains(username) && !data.requests_received.contains(username))
```

Comment:
I'm a bit confused about this.
So this list only shows players that you played but doesn't list the players you know ?

Reply:
Yes, the idea is to have the list of players you recently played with and you're not already friends with.